routes/movies.route.js

```diff
@@ -0,0 +1,66 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+const Movies = require("../models/Movies.model")
+
+const Celebrity = require("../models/Celebrity.model")
+
+// all your routes here
+
+router.get("/movies/create", async(req, res, next) => {
+    try {
+       
+        let celebrities = await Celebrity.find()
+        
+        res.render("movies/new-movie", {celebrities})
+
+    } catch(error) {
+        next(error)
+    }
+    });
+
+    router.post("/movies/create", async(req,res,next)=> {
+        try {
+            const {title, genre, plot, cast} = req.body
+    
+            await Movies.create({title, genre, plot, cast})
+            
+            res.redirect("/new-movie") 
```

Comment:
You don't have the route `/new-movie`, so you can't redirect there.

Remember that every time you use redirect, it has to be to a route that exists, ie:
`res.redirect('/')` || `res.redirect('/celebrities')`.

Here, it would make sense to redirect to `/movies`, and that's what the lab asked you to do. But the problem is that you still don't have that route done so you can't redirect there. While that, the best option would be to redirect to the main page `('/')`.